src/Tools/dotnet-monitor/Extensibility/ExtensionRepository.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.Diagnostics.Tools.Monitor.Extensibility
+{
+    internal abstract class ExtensionRepository : IExtensionRepository
+    {
+        private readonly int _resolvePriority;
+        private readonly string _name;
```

Comment:
Add comments to indicate name is a friendly name, not an identifier.

Reply:
renamed these to `DisplayName`